package.json

```diff
@@ -104,34 +104,11 @@
     "file-loader": "^1.1.11",
     "fork-ts-checker-webpack-plugin": "^0.4.15",
     "gulp": "^4.0.2",
-    "gulp-angular-filesort": "^1.2.1",
-    "gulp-angular-templatecache": "^2.2.7",
-    "gulp-autoprefixer": "^5.0.0",
-    "gulp-csso": "^3.0.1",
     "gulp-filter": "^5.1.0",
-    "gulp-flatten": "^0.4.0",
-    "gulp-header": "^2.0.9",
-    "gulp-htmlmin": "^4.0.0",
     "gulp-hub": "^4.2.0",
-    "gulp-inject": "^4.3.2",
-    "gulp-insert": "^0.5.0",
-    "gulp-jshint": "^2.1.0",
-    "gulp-load-plugins": "^1.6.0",
-    "gulp-minify-html": "^1.0.6",
-    "gulp-ng-annotate": "^2.1.0",
-    "gulp-rename": "^1.4.0",
-    "gulp-replace": "^1.0.0",
-    "gulp-rev": "^8.1.1",
-    "gulp-rev-replace": "^0.4.4",
-    "gulp-sass": "^4.0.2",
```

Comment:
❤️

Reply:
but gulp is used by severals script... `npm run lint`, `npm run build:prod`, `serve:prod`....

Reply:
I removed the `gulp-*` dependencies that appear not to be `require` anywhere (keeping only the minimal gulp deps for now) and everything else looks to be working (at least on my machine 😂 )

Reply:
ok it's my fault, i didn't read the error...

```
npm ERR! code ETARGET
npm ERR! notarget No matching version found for @gravitee/ui-components@^1.0.19-SNAPSHOT.
```